src/dotnet/commands/dotnet-new/FSharp_Console/project.json.template

```diff
@@ -9,31 +10,18 @@
       ]
     }
   },
-  "dependencies": {
-    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160509"
-  },
   "tools": {
-    "dotnet-compile-fsc": {
-      "version": "1.0.0-preview2-*",
-      "imports": [
-        "dnxcore50",
-        "portable-net45+win81",
-        "netstandard1.3"
-      ]
-    }
+    "dotnet-compile-fsc":"1.0.0-preview2-*"
```

Comment:
about that, next .net core sdk what will be? `preview3`? because if it's `preview3` it's better to use `1.0.0-preview3-*` so we dont break (by mistake) the current user of `preview2` if we need a new version.
we can publish `preview3` in a dev feed to test `dotnet-new` inside this repo (like other feeds)